src/store/actions.ts

```diff
@@ -223,7 +223,7 @@ export const clearErros = createAction(
 // FIXME: Move the rest to their own file
 export const setSettings = createAction(
   'SET_SETTINGS',
-  (settings: SettingsType) => {
+  (settings: Partial<SettingsType>) => {
```

Comment:
I'm OK with this change, though again, like many of my other comments, this is a very easy to miss change that is quite wide-reaching. Please make these in different PRs in the future, or at the very least different commits!